03-vectors/vectors-practice-2.Rmd

```diff
@@ -33,6 +33,16 @@ that showcases which ones have the best `mpg`, then, the ones that have the
 highest `hp`.  Using `order()`, rearrange this data frame such that it's ordered
 first by `mpg` (low to high) and then by `hp` (high to low).
 
+```{r}
+attach(mtcars)
+# mpg - low to high
+newdata <- mtcars[order(mpg),]
+newdata
+# hp - high to low
+newdata2 <- mtcars[order(hp, decreasing = TRUE),]
+newdata2
+```
+
```

Comment:
If you're going to attach you have to remember to detach otherwise it will get messy if you use another dataframe later with the same column names. I usually just do mtcars[order(mtcars$mpg),]